### DISCUSSION TITLE
What is the largest city in the 🇮🇹 region of Tuscany?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#994] [Galaxy Trivia] [answered]
**Bonus:** What dessert was invented there and who invented it?

**Comment by peterbaccaro** [comment]
The largest city in the Italian region of Tuscany is **Florence** (Firenze in Italian). Florence is not only the largest city in Tuscany but also one of the most famous and historically significant cities in Italy. It is renowned for its rich history, art, and architecture, including iconic landmarks such as the Duomo, the Uffizi Gallery, and the Ponte Vecchio.

The dessert invented in Florence is called **gelato**. While the origins of gelato can be traced back to ancient times, it was in Florence where the modern version of gelato was developed. The first inventor of the gelato was **Bernardo Buontalenti**, the architect of the Medici. Bernardo Buontalenti (Florence, 1531-1608) was a Florentine architect, theatrical designer, military engineer and artist.

**Comment by thuongtruong1009** [comment, marked as answer]
The largest city in the region of Tuscany, Italy, is Florence (Italian: Firenze). 

As for the dessert invented in Tuscany, the region is known for many culinary delights, but one of the most famous desserts associated with Tuscany is called "Tiramisu." However, Tiramisu is not specifically attributed to the region of Tuscany. It is a popular Italian dessert that originated in the Veneto region, specifically in the city of Treviso, near Venice. 

Tiramisu is made with layers of coffee-soaked ladyfingers (sponge biscuits) and a mixture of mascarpone cheese, eggs, and sugar, often dusted with cocoa powder. While the exact origins of Tiramisu are disputed, it is widely believed to have been created in the 1960s or 1970s. Various Italian restaurants and chefs claim to have invented it, but the most commonly mentioned name associated with its creation is Roberto Linguanotto, a pastry chef from Treviso. However, it's important to note that the exact details of its invention remain somewhat uncertain.
